tests/test_FLEbasis2D.py

```diff
@@ -70,8 +71,12 @@ def relerr(base, approx):
 
 @pytest.mark.parametrize("basis", test_bases, ids=show_fle_params)
 class TestFLEBasis2D(UniversalBasisMixin):
-    # Loosen the tolerance for `cufinufft` to be within 15%
-    test_eps = 1.15 if backend_available("cufinufft") else 1.0
+    # Loosen the tolerance for `cufinufft` and `osx_arm`
+    test_eps = 1.0
+    if backend_available("cufinufft"):
+        test_eps = 1.15
+    elif platform.system() == "Darwin":
+        test_eps = 1.20
```

Comment:
👍

Reply:
is this a concern to you?

Reply:
Not if it isn't to you. Are you ok with the 20%?

I was just giving a thumbs up to "fixing" it.

Reply:
Only thing that changed is the finufft .... not much I can do about it :/

Reply:
If they're not concerns let resolve them. Thanks